### ISSUE TITLE
Cloudflare Image Resizing SSG

### ISSUE THREAD

**Issue by alexanderniebuhr** [withastro/adapters#50]
### Astro Info

```block
*
```

### Describe the Bug

*

### What's the expected result?

*

### Link to Minimal Reproducible Example

https://google.de

### Participation

- [ ] I am willing to submit a pull request for this issue.

**Comment by gentksb** [comment]
Good, I want to use cloudflare image resizing w/ SSG

**Comment by gentksb** [comment]
[unpic](https://unpic.pics/) can resolve this.
This issue can be closed.

**Comment by alexanderniebuhr** [comment]
I don't think it can fully resolve the outlined requirements, but it's definitely a good solution.

**Comment by gentksb** [comment]
I got it. I tried some patterns and I still need fully support Cf images optimization.
